### PR TITLE
[map] a few small fixes

### PR DESCRIPTION
**Pull request by beets**
- added missing replot on prop update
- fix domain / color for wet bulb temp

small bugfix in timeline:
- removed empty measurement method in the provenance

### CODE REVIEW

**Comment by beets** [comment]
also updated tooltips / hovers:
- reduce opacity of non-selected regions, add shadow behind tooltips and update highlight border

image

**Comment by beets** [comment]
small update to scatter & map tooltips too (content-only update): 

image

File: static/js/chart/draw_choropleth.ts [hunk 1]
```diff
@@ -100,15 +100,23 @@ function getColorScale(
   const extent = d3.extent(Object.values(dataValues));
   const medianValue = d3.median(Object.values(dataValues));
   let domainValues: number[] = domain || [extent[0], medianValue, extent[1]];
-  if (statVar.indexOf("Temperature") >= 0) {
+  if (
+    statVar.indexOf("Temperature") >= 0 ||
```
Comment (shifucun) [inline static/js/chart/draw_choropleth.ts:104]: Nit: move this to the util function as a function like isTemperatureStatVar()?

File: static/js/chart/draw_choropleth.ts [hunk 2]
```diff
@@ -100,15 +100,23 @@ function getColorScale(
...
       d3.interpolateBlues(0.8),
       MIN_COLOR,
       d3.interpolateReds(0.8),
       d3.interpolateReds(1),
     ];
-    if (statVar.indexOf("Difference") >= 0) {
+
+    if (statVar.indexOf("Number of Months Based on") >= 0) {
+      // SV Name of "Wet Bulb Temps"
+      domainValues = domain || [0, 15];
```
Comment (chejennifer) [inline static/js/chart/draw_choropleth.ts:118]: nit: these curated domains (eg. [0, 15], [-10, -5, 0, 5, 10]) could be constants

File: static/js/chart/draw_choropleth.ts [hunk 3]
```diff
@@ -525,6 +533,7 @@ const onMapClick = (
 
 function mouseOutAction(domContainerId: string, index: number): void {
   const container = d3.select(domContainerId);
+  container.classed(HOVER_HIGHLIGHTED_CLASS_NAME, false);
```
Comment (chejennifer) [inline static/js/chart/draw_choropleth.ts:536]: duplicate of line 539

File: static/js/chart/draw_choropleth.ts [hunk 4]
```diff
@@ -539,6 +548,7 @@ function mouseHoverAction(
   const container = d3.select(domContainerId);
   // show highlighted border and show cursor as a pointer
   if (canClick) {
+    container.classed(HOVER_HIGHLIGHTED_CLASS_NAME, true);
```
Comment (chejennifer) [inline static/js/chart/draw_choropleth.ts:551]: duplicate of line 555

**Review by chejennifer — APPROVED** [review]

File: static/js/chart/draw_choropleth.ts (hunk 3 above)
Comment (beets) [reply static/js/chart/draw_choropleth.ts:536]: on purpose - setting the class to the container as well so other regions are made more transparent

File: static/js/chart/draw_choropleth.ts (hunk 4 above)
Comment (beets) [reply static/js/chart/draw_choropleth.ts:551]: ditto

File: static/js/chart/draw_choropleth.ts (hunk 2 above)
Comment (beets) [reply static/js/chart/draw_choropleth.ts:118]: done

File: static/js/chart/draw_choropleth.ts (hunk 1 above)
Comment (beets) [reply static/js/chart/draw_choropleth.ts:104]: done

**Pull request merged**